graffiti/graph/traversal/traversal_test.go

```diff
@@ -803,3 +810,47 @@ func TestLimit(t *testing.T) {
 		t.Fatalf("Should return 1 result, returned: %v", res.Values())
 	}
 }
+
+func TestDedupMultiplefields(t *testing.T) {
+	g := newGraph(t)
+
+	g.NewNode(graph.GenID(), graph.Metadata{"Name": "aaa", "Type": "intf", "Value": "v1"})
+	g.NewNode(graph.GenID(), graph.Metadata{"Name": "bbb", "Type": "intf", "Value": "v1"})
+	g.NewNode(graph.GenID(), graph.Metadata{"Name": "aaa", "Type": "veth"})
+	g.NewNode(graph.GenID(), graph.Metadata{"Name": "aaa", "Type": "intf"})
+
+	// next traversal test
+	query := `G.V().Dedup("Name")`
+	res := execTraversalQuery(t, g, query)
+	if len(res.Values()) != 2 {
+		t.Fatalf("Should return 2 nodes, returned: %v", res.Values())
+	}
+
+	// next traversal test
+	query = `G.V().Dedup("Name", "Type")`
+	res = execTraversalQuery(t, g, query)
+	if len(res.Values()) != 3 {
+		t.Fatalf("Should return 3 nodes, returned: %v", res.Values())
+	}
+
+	// next traversal test
+	query = `G.V().Dedup("Name", "Type")`
```

Comment:
It seems to be the same test than the previous one

Reply:
Yes but I really want to be sure !